sample/development/components/integration/test/integrationTests.brs

```diff
@@ -521,7 +521,7 @@ function TS_SDK_integration() as object
                 ADB_assertTrue((debugInfo.configuration.edge_configid <> invalid and Len(debugInfo.configuration.edge_configid) > 10), LINE_NUM, "assert edge_configid is valid")
                 ADB_assertTrue((debugInfo.configuration.media_channel = "channel_test"), LINE_NUM, "assert media_channel is valid")
                 ADB_assertTrue((debugInfo.configuration.media_playerName = "player_test"), LINE_NUM, "assert media_playerName is valid")
-                ADB_assertTrue((debugInfo.configuration.media_appVersion = "1.0.0"), LINE_NUM, "assert media_appVersion is valid")
+                ADB_assertTrue((debugInfo.configuration.media_appVersion = "1.1.0-alpha"), LINE_NUM, "assert media_appVersion is valid")
```

Comment:
can we have test-constant for this so we dont need to update the tests everywhere.

Reply:
I prefer to keep those raw values in tests. It's easy to update those values when bumping the SDK version, and it's obvious when checking the test code.  Also, it's not straight forward to add a "constant" in Brightscript and to be referred to in multiple files.